packages/react-atlas-default-theme/webpack.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     rules: [
     {
       test: /\.css$/,
+      exclude: /\.global\.css$/,

```

Comment:
Some classes are just hard code in children components of bs-table like .react-bs-table-pagination or .select-filter etc, css loader couldn't be used in this case. so I need to use global css without modulizing to apply class styles which are inside bs-table. 

Do you have suggestions to accomplish this? I figured out to handle this situation for a while. I know it's not a good practice to mix local/global but has no other solution for now.

Some discussions: https://github.com/css-modules/css-modules/pull/65